### DISCUSSION TITLE
1000 API calls limit

### DISCUSSION THREAD

**Discussion by Hellis81** [ekutner/home-connect-hass#88] [Ideas]
I have an idea. 
The Home connect developer tools allow you to have multiple applications, each with 1000 API calls.
So, what if the integration has two home connect client ids, and secrets? 
When time is between midnight and noon, use application 1, then use application 2.

Or have the integration use n number of client ids and secrets and just do a 86400/n seconds when to switch between the applications.
And those who has massive amount of appliances or just love washing clothes can use the integration all day.
Wouldn't this be possible?

**Comment by ekutner** [comment]
This has already been discussed here #60, tl;dr I'm not aware of a way to do it with HA.
